rules/no-object-as-default-parameter.js

```diff
@@ -4,6 +4,8 @@ const getDocumentationUrl = require('./utils/get-documentation-url');
 const MESSAGE_ID = 'noObjectAsDefaultParameter';
 
 const objectParameterSelector = [
+	':matches(FunctionDeclaration, ArrowFunctionExpression)',
```

Comment:
I think `:function` should work.

Reply:
If all your tests passed, you must missed the `FunctionExpression`, pleaseadd one.

Reply:
Yep, I've missed that one. Fixed.